### ISSUE TITLE
alleles in Z score file

### ISSUE THREAD

**Issue by alessandrotestori** [huwenboshi/ImpG#4]
Hello!
I have noticed that while running ImpG there might be problems when alleles in my input Z-score file (ZF) differ from the provided mapping file (MF). Am I right?
  Basically, I have four cases:
1 - Alt allele and Ref allele match in my ZF and in the MF;
2 - Alt allele and Ref allele are inverted in my ZF with respect to the MF (e.g.: Alt allele in MF = A; Ref allele in MF = G; Alt allele in ZF = G; Ref allele in ZF = A);
3 - Alt allele and Ref allele in my ZF are the complement of Alt allele and Ref allele in the MF (e.g.: Alt allele in MF  = A; Ref allele in MF = G; Alt allele in ZF = T; Ref allele in ZF = C);
4 - Alt allele and Ref allele in my ZF are inverted and the complement of Alt allele and Ref allele in the MF (e.g.: Alt allele in MF  = A; Ref allele in MF = G; Alt allele in ZF = C; Ref allele in ZF = T);

If I understand correctly, I can still use ImpG, provided that I make the alleles in my ZF consistent with the alleles in the MF. When flipping or inverting, however, I also should invert z-scores in some cases (case 2 and 4): am I right?

Please, let me know. Thanks, best regards

**Comment by huwenboshi** [comment]
Yes, you are correct. Also, you should also filter out strand-ambiguous SNPs, e.g. SNPs with reference and alternative alleles (A, T) or (C, G).
